.github/renovate.json5

```diff
@@ -419,6 +419,7 @@
   "regexManagers": [
     {
       "fileMatch": [
+        "images/cilium/download-hubble\\.sh",
```

Comment:
By the way if the json format allows comments, we could add a comment above to remove this following Cilium 1.18 release.